ACadSharp/Entities/MText.Font.cs

```diff
@@ -0,0 +1,138 @@
+﻿#nullable enable
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace ACadSharp.Entities
+{
+	public partial class MText
+	{
+		/// <summary>
+		/// Font used by MText Tokens.
+		/// </summary>
+		public class Font
+		{
+			/// <summary>
+			/// Tet Font Family.
+			/// </summary>
+			public ReadOnlyMemory<char> FontFamily { get; set; }
+
+			/// <summary>
+			/// Bold Text
+			/// </summary>
+			public bool IsBold { get; set; }
+
+			/// <summary>
+			/// Italic text.
+			/// </summary>
+			public bool IsItalic { get; set; }
+
+			/// <summary>
+			/// Code page for the text.
+			/// </summary>
+			public int CodePage { get; set; }
+
+			/// <summary>
+			/// Text pitch.
+			/// </summary>
+			public int Pitch { get; set; }
+
+			/// <summary>
+			/// Creates a blank font.
+			/// </summary>
+			public Font()
+			{
+
+			}
+
+			/// <summary>
+			/// Creates a font with the contents of the passed font.
+			/// </summary>
+			/// <param name="original">Original font to copy from.</param>
+			public Font(Font original)
+			{
+				this.OverrideFrom(original);
+			}
+
+			/// <summary>
+			/// Creates a font with the passed formats.  Used for testing.
+			/// </summary>
+			/// <param name="formats">Formats to use.</param>
+			internal Font(string formats)
```

Comment:
I think is a bad practice to add methods in the main code only for testing purposes, it may be more tedious but this methods should be implemented in some other way in the `test` project.

In this particular case I can see that this is not being used and it should be removed.